docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -163,6 +202,16 @@ \section{Safety and Security Requirements}
 
 \wss{Newly discovered requirements.  These should also be added to the SRS.  (A
 rationale design process how and why to fake it.)}
+\begin{enumerate}[{SFR}1.]
```

Comment:
what does SFR mean? I don't see that as a section in our requirements document

Reply:
Safety Requirement, figured that section would need the requirements numbered as they are in the SRS

Reply:
Yeah they should be, but my confusion is over the label, because we didn't use SFR as the label for safety requirements in the SRS

Reply:
Probably be consistent to avoid confusion

Reply:
I don't believe there is a safety requirements section in the SRS? There's a safety-critical section in 12 performance requirements (PR-SC#), but that's only for performance requirements.****

Reply:
we can put it as an access requirement

Reply:
That works, so we'll disregard a safety requirements label such as SFR# in HazardAnalysis.pdf and just use what number we will give it in SRS.pdf? (for this one it will be SR-AR5).